transport/x/grpc/inbound.go

```diff
@@ -85,22 +91,17 @@ func (i *Inbound) start() error {
 	if i.router == nil {
 		return errRouterNotSet
 	}
-	serviceDescs, err := i.getServiceDescs()
-	if err != nil {
-		return err
-	}
+
+	i.handler = newHandler(
```

Comment:
Not particularly, I did it mostly for consistency with the existing handler code.